store/set.js

```diff
@@ -54,22 +54,37 @@ export const actions = {
       .then(() => {
         return this.$apis.set.modifyItems('add', state.likesId, itemId)
           .then(commit('like', itemId));
+      })
+      .catch(() => {
+        return dispatch('fetchLikes');
       });
   },
-  unlike({ commit, state }, itemId) {
+  unlike({ dispatch, commit, state }, itemId) {
     return this.$apis.set.modifyItems('delete', state.likesId, itemId)
-      .then(commit('unlike', itemId));
+      .then(commit('unlike', itemId))
```

Comment:
I think this can be reduced to:
```js
      .then(dispatch('fetchLikes'))
      .catch(dispatch('fetchLikes'));
```
... which is more readable. Likewise the other changes to this file.

Reply:
That breaks the error handling again. 'then' and 'catch' expect callbacks. I propose to leave as is.

Reply:
Indeed. Thanks for the sanity check @LeoniePeters !